content/upgrade-pop.md

```diff
@@ -87,19 +89,21 @@ sudo apt update
 sudo apt full-upgrade
 ```
 
-Once the updates are applied, a notification should appear at the top of your screen saying that an upgrade is available. Click on this notification, or go to Settings ->  OS Upgrade & Recovery. The System76 upgrade package will display a message that Pop!\_OS 21.10 is available with a `Download` button.
+Once the updates are applied, a notification should appear at the top of your screen saying that an upgrade is available. Click on this notification, or go to Settings ->  OS Upgrade & Recovery. The System76 upgrade package will display a message that Pop!\_OS 22.04 is available with a `Download` button.
 
 If you are planning on staying on an LTS release for the time being, this is also the page where you can dismiss upgrade notifications.
 
-![Settings OS Upgrade](/images/upgrade-pop/setting-os-upgrade.png)
+The OS Upgrade and Recovery page of settings now allows for automatic update scheduling and updating the Recovery partition as well.
+
+![Settings OS Upgrade](/images/upgrade-pop/Pop-22.04-Upgrade.png)
```

Comment:
It's fine if this filename changes, but the old image should be removed if it's no longer in use.

Reply:
Removed.